Firmware/temperature.cpp

```diff
@@ -2676,7 +2676,7 @@ static bool autotune(int16_t cal_temp)
         set_fan_speed(speed);
         wait(10000);
 
-        sprintf_P(tm_message, PSTR("TM: R[%u] estimat."), (unsigned)i);
+        sprintf_P(tm_message, PSTR("TM: R[%u] estimate."), (unsigned)i);
```

Comment:
Looks good to me.

`i` is `int8_t` so is maximum 3 characters.
```
Character Count: |0|1|2|3      |4|5|6|7|8|9|0      |1|2|3|4|5|6|7|8|9|
String:          |T|M|:|<space>|R|[|1|2|3|]|<space>|e|s|t|i|m|a|t|e|.|
```

And tm_message buffer is 21 bytes, where the last byte is null for string termination👍

Reply:
Thanks for the review and approval.